rawfile/rawfile_servicer.py

```
@@ -221,6 +222,11 @@ def CreateVolume(self, request, context):
             else:
                 raise exc
 
+        def volume_provision(usage: Usage):
+            usage.volume_provision(request.name, size)
+
+        send_event(volume_provision)
```

in `send_event` calls should we ignore errors, So CSI call wont don't get error?

---

There should not be any errors on `send_event` since we only push to the queue?

---

Oh, I have not seen that